packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -183,6 +184,11 @@ export class Fixture {
       return origCreateElement(type, props, ...children);
     });
 
+    //eslint-disable-next-line jasmine/no-unsafe-spy
+    spyOn(BlurHash, 'default').and.callFake(() => ({
```

Comment:
Good lookin out, everything is passing now! I got too deep in on the wrong path and thought the wrong thing was timing out.